explorer.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"html/template"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/decred/dcrutil"
+
+	apitypes "github.com/dcrdata/dcrdata/dcrdataapi"
+	"github.com/decred/dcrd/dcrjson"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+	"github.com/rs/cors"
+)
+
+type explorerMux struct {
+	*chi.Mux
+}
+
+func (c *appContext) explorerUI(w http.ResponseWriter, r *http.Request) {
+	helpers := template.FuncMap{
+		"getTime": getTime,
+	}
+	explorerTemplate, _ := template.New("explorer").Funcs(helpers).ParseFiles("views/explorer.tmpl", "views/extras.tmpl")
+
+	idx := c.BlockData.GetHeight()
+	N := 25
+	start, errS := strconv.Atoi(r.URL.Query().Get("startat"))
+	if errS != nil || start == 0 {
+		start = idx - N + 1
+	}
```

Comment:
Noticed browser crashes when I ask for large range, probably simply can't render that much html.
Maybe add a pagesize param that defaults to N?

Reply:
yes, me too.  there needs to be a limit.  The API request for 10k blocks takes less than 10 seconds (still a alot), but the browser just can't handle it.

Reply:
Some way to control parameters `start=A&end=B` would perhaps be better than specifying one end and having the other end be best block height.

Reply:
hmm, I was thinking start<end, but I suppose if you consider start is at the top of the page, I can see why start would be the larger number.  But IMO it does seem counter-intuitive to do `start=100&end=0` and not the other way around.